practice/src/assignment1/Stack.kt

```diff
@@ -0,0 +1,22 @@
+package assignment1
+
+class Stack<T> {
+
+    private var stack = ArrayList<T>()
+
+    fun push(element: T) {
+        stack.add(element)
+    }
+    fun pop(): T? {
+        if(stack.isEmpty()) {
+            return null
+        }
+        return stack.removeAt(stack.size - 1)
```

Comment:
```suggestion
        return stack.removeLast()
```

Kotlin Collections API를 활용해서 조금 더 쉽게 코드를 작성할 수 있습니다.